whois-api/src/main/java/net/ripe/db/whois/api/rdap/domain/RdapObject.java

```diff
@@ -117,6 +119,13 @@ public List<Autnum> getAutnums() {
         return autnums;
     }
 
+    public List<Redaction> getRedacted() {
+        if (redacted == null) {
+            redacted = Lists.newArrayList();
```

Comment:
return this.reacted == null ? Lists.newArrayList() : redacted;

Reply:
I can't really explained but if I do this
rdapObject.getRefacted().addAll(x) 
Doesn't add X to the list. So I am leaving as it is, and in that Object we are doing the same for rest of attributes.
We are not using the short statement